types/gm/gm-tests.ts

```diff
@@ -308,7 +309,9 @@ gm(src)
 	.format({ bufferStream: true }, (err, format) => {
 	})
 	.identify((err, info) => {
-	})
+    })
+    .identify(customFormat, (err, info) => {
+    })
```

Comment:
Nitpick but could you fix the extra space indent?

Reply:
Should be fixed now.

Reply:
Awesome!